examples/language-sdk-instrumentation/golang-push/rideshare/docker-compose.yml

```diff
@@ -1,5 +1,6 @@
 services:
   us-east:
+    container_name: us-east
```

Comment:
I'd say this isn't needed 🤔

Reply:
its needed so i can add them in the prometheus.yml file without having to use the funky autogenerated name docker gives when not adding it